apps/condo/domains/notification/constants.js

```diff
@@ -28,6 +29,10 @@ const MESSAGE_META = {
         userPhone: { defaultValue: '', required: false },
         organizationName: { defaultValue: 'ORGANIZATION', required: false },
     },
+    [DIRTY_INVITE_NEW_EMPLOYEE_MESSAGE_TYPE]: {
```

Comment:
We have `DIRTY_INVITE_NEW_EMPLOYEE_MESSAGE_TYPE`, `ReInviteOrganizationEmployeeInput`, `reInvite`, `RetryInviteMessage`…, — all the things are about the same.
Maybe, it will make this feature easier to figure out, when name will be consistent, like:

`RETRY_INVITE_NEW_EMPLOYEE_MESSAGE_TYPE`, `RetryInviteOrganizationEmployeeInput`, `retryInvite`, `RetryInviteMessage`.

"Retry…" prefix seems to be good.